### PR TITLE
Update working-with-the-input-component.md

### LINKED ISSUE #945: 'Working with the input component' documentation is out of date
**Describe the bug**
The page https://o3de.org/docs/user-guide/interactivity/input/working-with-the-input-component/ features outdated instructions.

**To Reproduce**
1. Go to https://o3de.org/docs/user-guide/interactivity/input/working-with-the-input-component/
2. Follow the instructions to setup the script
3. Observe the script does not work

**Expected behavior**
The proposed script fails due to the lack of an OnEntityActivatedNode to feed into the InputHandler node. The documentation should be updated to respect this.

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows

### LINKED ISSUE #951: Docs Images Outdated for Input Component
**Describe the bug**
The button icon has changed making the doc slightly confusing.
https://o3de.org/docs/user-guide/interactivity/input/working-with-the-input-component/

image

### PR DESCRIPTION
**Pull request by FiniteStateGit**
Fix #951; Fix #945

### CODE REVIEW

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+You can also create a Lua script that connects to Input Events. For more information about scripting with Lua, refer to [Writing Lua Scripts](/docs/user-guide/scripting/lua).
+
+1. In Perspective, select the entity.
+
+1. In Entity Inspector, click **Add Component**, and add the **Lua Script** component.
+
+1. In the Lua Script component, create a table of the functions `OnPressed`, `OnHeld`, and `OnReleased` for an Input Event you would like to connect the script to.
+
+1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the Input Event, and the second parameter set to create an InputEventNotificationId for the **Event Name**.
```
Comment (willihay) [inline content/docs/user-guide/interactivity/input/using-player-input.md:86]:
```suggestion
1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the Input Event, and the second parameter set to create an `InputEventNotificationId` for the **Event Name**.
```

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+end
+
+return tutorial_input
+```
+
+## Testing player input
+
+After you use either Script Canvas or Lua to connect to an Input Handler for the Input Event in your `.inputbindings` file, you can test player input in **O3DE Editor**.
+
+1. In Perspective, select your entity.
```
Comment (willihay) [inline content/docs/user-guide/interactivity/input/using-player-input.md:125]:
```suggestion
1. In the viewport or **Entity Outliner**, select your entity.
```

**Review by willihay — CHANGES_REQUESTED** [review]
Nice update! Just a handful of suggestions.

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+### Using input in Lua 
+
+You can also create a Lua script that connects to Input Events. For more information about scripting with Lua, refer to [Writing Lua Scripts](/docs/user-guide/scripting/lua).
+
+1. In Perspective, select the entity.
+
+1. In Entity Inspector, click **Add Component**, and add the **Lua Script** component.
+
+1. In the Lua Script component, create a table of the functions `OnPressed`, `OnHeld`, and `OnReleased` for an Input Event you would like to connect the script to.
```
[the remark this answers is not in the document]
Comment (FiniteStateGit) [reply content/docs/user-guide/interactivity/input/using-player-input.md:84]: Good catch!

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+1. In the viewport or **Entity Outliner**, select the entity.
+
+1. In Entity Inspector, click **Add Component**, and add the **Lua Script** component.
+
+1. In the Lua Script component, click the **Open in Lua Editor** icon.
+
+1. In the Lua Editor, create a table of the functions `OnPressed`, `OnHeld`, and `OnReleased` for an Input Event you would like to connect the script to.
+
+1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the Input Event, and the second parameter set to create an `InputEventNotificationId` for the **Event Name**.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:92]:
```suggestion
1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the input event, and the second parameter set to create an `InputEventNotificationId` for the **Event Name**.
```
Review the wording of this step. As currently written, I can't make sense of it. (There might be some stray prepositions adding to my confusion.) If possible, break it into two sentences for additional clarity.

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+function tutorial_input:OnDeactivate()
+    self.ActionInputHandler:Disconnect()
+end
+
+return tutorial_input
+```
+
+## Testing player input
+
+After you use either Script Canvas or Lua to connect to an Input Handler for the Input Event in your `.inputbindings` file, you can test player input in **O3DE Editor**.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:129]:
```suggestion
After you use either Script Canvas or Lua to connect to an input handler for the input event in your `.inputbindings` file, you can test player input in **O3DE Editor**.
```
* Made "Input Handler" lower case. Outside of the UI and script references to "InputHandler", there doesn't seem to be a reason to capitalize general references to this concept.
* The following rewording sounds a lot better to my ears. Would it be accurate?
  CURRENT
  >connect to an input handler for the input event in your `.inputbindings` file
  
  REVISED
  >connect the input event in your `.inputbindings` file to an input handler

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+1. In the viewport or **Entity Outliner**, select your entity.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:131]: _Same comments as for earlier procedures_
* What UI surface is this? Clarify.
* "select your entity"
  * Will the reader know what we mean here by "the entity"? Is this the `.inputbindings` file? Clarify.

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+
+    {{< note >}}
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:133]:
```suggestion
{{< note >}}
```

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+In this example, the entity must have a component that enables visibility in the game, such as the [Mesh component](/docs/user-guide/components/reference/atom/mesh), the [White Box component](/docs/user-guide/components/reference/shape/white-box), or one of the [Shape components](/docs/user-guide/components/reference/shape).
+{{< /note >}}
+
+1. In Entity Inspector, click **Add Component**, and add the **Mesh** component.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:137]:
```suggestion
1. In Entity Inspector, click **Add Component**, and then add the **Mesh** component.
```

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+
+1. For **Mesh asset**, specify a mesh asset file. This gives your entity a shape that is visible in game mode.
+
+1. Press **Ctrl+G** to enter game mode.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:141]:
```suggestion
1. To enter game mode, press **Ctrl+G**.
```

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+
+1. Press the keyboard **Spacebar** so that your entity rotates on the local z-axis. The entity's scale doubles when the event `OnPressed` is received from the Input Handler that is connected to `Action`.  The entity's scale returns to normal when the `OnReleased` event is received.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:143]:
```suggestion
1. Press **Space bar** so that your entity rotates on the local z-axis. The entity's scale doubles when the event `OnPressed` is received from the input handler that is connected to `Action`. The entity's scale returns to normal when the `OnReleased` event is received.
```
To avoid passive voice ("is received"), can we identify who/what receives the events? For example:
>The entity's scale doubles when [noun] receives the `OnPressed` event from the input handler that is connected to `Action`. The entity's scale returns to normal when [noun] receives the `OnReleased` event.

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+
+1.  To exit gameplay mode, press **Esc**.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:145]:
```suggestion
1.  To exit game mode, press **Esc**.
```

**Review by paucom — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/interactivity/input/using-player-input.md (hunk 7 above)
Comment (FiniteStateGit) [reply content/docs/user-guide/interactivity/input/using-player-input.md:133]: Here, the indentation is needed to avoid splitting the ordered list.

File: content/docs/user-guide/interactivity/input/using-player-input.md (hunk 10 above)
Comment (FiniteStateGit) [reply content/docs/user-guide/interactivity/input/using-player-input.md:143]: Thanks for mentioning this too, I was unsure how to treat Script Canvas nodes. The capitalization of Input Handler was mean to convey that it is a node.  I'll correct the passive voice; any thoughts on formatting for SC nodes though?

File: content/docs/user-guide/interactivity/input/using-player-input.md (hunk 5 above)
Comment (FiniteStateGit) [reply content/docs/user-guide/interactivity/input/using-player-input.md:129]: I'll incorporate both of these edits, thanks for taking the time, I write everything down for future reference.

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+
+To create an `.inputbindings` file with Asset Editor, follow these steps.
+
+1. In **O3DE Editor**, create an entity.
+
+1. Select the entity in the viewport or **Entity Outliner**.
+
+1. In **Entity Inspector**, click **Add Component**, and then add an Input component.
+
+1. In Entity Inspector, find the Input component, and then click **Open in Asset Editor** to create a new `.inputbindings` file in Asset Editor.
```
[the remark this answers is not in the document]
Comment (FiniteStateGit) [reply content/docs/user-guide/interactivity/input/using-player-input.md]: Awesome, added both of those new icons in.

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+
+1. In O3DE Editor, select the entity with the Input component in the viewport or **Entity Outliner**.
+
+1. In Entity Inspector, click **Add Component** and add the **Script Canvas** component.
+
+1. In the Script Canvas component, for **Script Canvas Asset**, click {{< icon "open-in-internal-app.svg" >}}, and then create a new Script Canvas graph like the following example.
+
+### Example Script Canvas graph
+
+In the following graph, the **InputHandler** node connects `Action` events to various **Transform** nodes. The Input component sends **Pressed**, **Held**, and **Released** events to the **InputHandler** node When the state of an event generator changes.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:76]:
```suggestion
In the following graph, the **InputHandler** node connects `Action` events to various **Transform** nodes. When the state of an event generator changes, the Input component sends **Pressed**, **Held**, and **Released** events to the **InputHandler** node.
```
Typo: Capital "When" in the middle of the sentence.

I still recommend moving the conditional phrase ("When the state...") to the front of the second sentence, where capitalizing it is fine.

If you prefer to keep that phrase at the end, be sure to make "when" lower case.

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+In this example, the entity must have a component that enables visibility in the game, such as the [Mesh component](/docs/user-guide/components/reference/atom/mesh), the [White Box component](/docs/user-guide/components/reference/shape/white-box), or one of the [Shape components](/docs/user-guide/components/reference/shape).
+{{< /note >}}
+
+1. In Entity Inspector, click **Add Component**, and then add the **Mesh** component.
+
+1. For **Mesh asset**, specify a mesh asset file. This gives your entity a shape that is visible in game mode.
+
+1. To enter game mode, press **Ctrl+G**.
+
+1. Press **Space bar** so that your entity rotates on the local z-axis. The entity's scale doubles when the event `OnPressed` is received from the input handler that is connected to `Action`. The entity's scale returns to normal when the `OnReleased` event is received.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:143]: Following up from [our previous discussion](https://github.com/o3de/o3de.org/pull/998#discussion_r728519987), try to reword this to avoid passive voice ("is received").

>any thoughts on formatting for SC nodes though?

To refer to a specific SC node, I'd go with "**[NodeName]** node", as you did in line 76 ("the **InputHandler** node").

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+
+### Setting event generator properties
+
+The following `.inputbindings` file specifies a keyboard for the **Input Device Type** and the space bar for the **Input Name**.
+
+   ![Input bindings configuration example in Asset Editor.](/images/user-guide/interactivity/input/inputbindings-example.png)
+
+### Saving your input bindings asset
+
+1. In the **Asset Editor**, choose **File, Save**.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:54]:
```suggestion
1. In the **Asset Editor**, choose **File**, **Save**.
```

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+
+1. Enter a name for the `.inputbindings` file, and then choose **Save**.
+
+1. In Entity Inspector, in the Input component, for **Input to event bindings**, click {{< icon "browse-edit-select-files.svg" >}} and select your `.inputbindings` file.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:58]:
```suggestion
1. In Entity Inspector, in the Input component, for **Input to event bindings**, click {{< icon "browse-edit-select-files.svg" >}}, and then select your `.inputbindings` file.
```

File: content/docs/user-guide/interactivity/input/using-player-input.md [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+function tutorial_input:OnDeactivate()
+    self.ActionInputHandler:Disconnect()
+end
+
+return tutorial_input
+```
+
+## Testing player input
+
+After you use either Script Canvas or Lua to connect the input event in your `.inputbindings` file to an input handler, you can test player input in **O3DE Editor**.
```
Comment (paucom) [inline content/docs/user-guide/interactivity/input/using-player-input.md:129]:
```suggestion
After you use either Script Canvas or Lua to connect the input event in your `.inputbindings` file to an input handler, you can test player input in O3DE Editor.
```
This occurrence of "O3DE Editor" is well after the first usage, so it doesn't need bold formatting.

**Review by paucom — CHANGES_REQUESTED** [review]
A few final edits

**Review by paucom — APPROVED** [review]
